locale/de_DE/LC_MESSAGES/djangojs.po

```diff
@@ -190,6 +190,51 @@ msgstr "Bearbeiten"
 msgid "Report"
 msgstr "Melden"
 
+#: adhocracy4/comments_async/static/comments_async/ai_report.jsx:6
+#: adhocracy4/comments_async/static/comments_async/ai_report.jsx:6
+msgctxt "defakts"
+msgid ""
```

Comment:
hmmm should we delete the translations for defakts? They will just confuse translators for mb